.goreleaser.yml

```
@@ -0,0 +1,34 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+brews:
+  - github:
+      owner: ggilmore
```

```suggestion
      owner: slimsag
```

You'll want to create slimsag/homebrew-update-docker-tags